english/add_music.md

```diff
@@ -0,0 +1,11 @@
+Add Music for your cake recipe?
```

Comment:
@KarenWest, I had never created a directory on the GitHub website before. I always clone and make all changes locally. However, I found [this post](http://stackoverflow.com/questions/18773598/creating-folders-inside-github-com-repo-without-using-git) with instructions on how to create a directory on GitHub. It's not a very intuitive way, but I just tried it in a test repository and it worked for me. Of course, cloning and creating the directory on your own computer is a great way to get more practice with remotes and pushing.